package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This should be in `tests/Application`, not in the root of the plugin, where it is useless anyway.

Compare this file, injected by Flex, with the old one in `tests/Application` and decide if there are some changes we should copy (or if we should simply replace old one with this one? :thinking: ) - try and test by `yarn install` and `yarn build`, or simply `make ci`